exporter/elasticsearchexporter/factory.go

```diff
@@ -129,6 +143,11 @@ func createMetricsExporter(
 	cf := cfg.(*Config)
 	logConfigDeprecationWarnings(cf, set.Logger)
 
+	// Workaround to avoid rejections from Elasticsearch
+	// TSDB does not accept 2 documents with the same timestamp and dimensions
+	cf.BatcherConfig.MaxSizeConfig.MaxSizeItems = 0
+	set.Logger.Info("batcher.max_size_items is ignored: metrics exporter does not support batch splitting")
```

Comment:
[to reviewer] I can change this to a warn, and it'll require some changes to tests as well. But let's first confirm if overriding max_size_items here look fine

Reply:
Warn sounds good.